packages/js-sdk/src/sandbox/sandboxApi.ts

```diff
@@ -80,6 +85,52 @@ export class SandboxApi {
     return true
   }
 
+  /**
+   * Get running sandbox.
+   *
+   * @param sandboxId sandbox ID.
+   * @param opts connection options.
+   *
+   * @returns running sandbox.
```

Comment:
doesn't this just return `SanboxInfo` ? My understanding is that `connect()` returns an actual instance of `Sandbox`, in which case, shouldn't the method be called `getInfo` or something to avoid confusion?